src/frontend/data/sideEffects/pollForTrack/index.tsx

```diff
@@ -46,3 +49,11 @@ export async function pollForTrack<
     return requestStatus.FAILURE;
   }
 }
+
+const isReadyToPlay = (object: Document | Video | TimedText): boolean => {
```

Comment:
Maybe this can be done in another PR but this seems to me like something that should be harmonized between object types and not abstracted away behind a method.

`is_ready_to_show` would work for basically anything we want to do with Marsha?

Reply:
:+1: